Package.swift

```diff
@@ -17,7 +17,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/bitmovin/player-ios.git", from: "3.100.0"),
+        .package(url: "https://github.com/bitmovin/player-ios.git", exact: "3.103.0"),
```

Comment:
Why not `from` anymore?

```suggestion
        .package(url: "https://github.com/bitmovin/player-ios.git", from: "3.103.0"),
```